testfixtures/logcapture.py

```diff
@@ -1,12 +1,26 @@
 from collections import defaultdict
+import os
 import atexit
 import logging
 import warnings
 from pprint import pformat
-
 from testfixtures.comparison import compare
 from testfixtures.utils import wrap
 
+_FAIL_ON_ERROR_LOG = os.environ.get("TEST_FIXTURES_FAIL_ON_ERROR_LOG") == "True"
```

Comment:
Nothing else is controlled by environment variables and I don't want to start that pattern, so let's lose this.
If you want it to be "on everywhere", just have a subclass or partial in your project.

Reply:
I would like to be able to enable strict on existing uses of LogCapture without code modifications...that is the point of this.
I can go the path of creating utility wrappers in my code, but from a user perspective it is not great.
Env var might not be the right way to tweak behaviors, is there any other acceptable way?